example/Tests/worklet-tests.ts

```diff
@@ -197,6 +197,35 @@ export const worklet_tests = {
     });
     return ExpectValue(result, 42);
   },
+  check_jsi_object_is_spreadable_after_worklet: async () => {
+    const fw = () => {
+      "worklet";
+      return { a: 100 };
+    };
+    let wf = Worklets.defaultContext.createRunAsync(fw);
+    const result = await wf();
+    const spreadObject = { ...result };
+    return ExpectValue(spreadObject, { a: 100 });
+  },
+  check_jsi_object_is_assignable_after_worklet: async () => {
+    const fw = () => {
+      "worklet";
+      return { a: 100 };
+    };
+    let wf = Worklets.defaultContext.createRunAsync(fw);
```

Comment:
Also we can just do `runAsync(func)` instead of the create stuff. It's less code.

Reply:
Changed it so